homeassistant/components/netatmo/light.py

```diff
@@ -47,6 +49,20 @@ def _create_camera_light_entity(netatmo_device: NetatmoDevice) -> None:
         )
     )
 
+    @callback
+    def _create_entity(netatmo_device: NetatmoDevice) -> None:
+        try:
+            getattr(netatmo_device.device, "brightness")
+        except AttributeError:
+            return
```

Comment:
Is this a stylistic thing or would hasattr solve this as well?
```suggestion
        if not hasattr(netatmo_device.device, "brightness"):
            return
```